src/astro/sql/operators/agnostic_boolean_check.py

```diff
@@ -7,7 +7,7 @@
 from sqlalchemy.sql.selectable import Select
 
 from astro.databases import create_database
-from astro.sql.operators.sql_decorator import SqlDecoratedOperator
+from astro.sql.operators.sql_decorator_legacy import SqlDecoratedOperator
```

Comment:
We can remove the checks for now for 0.9.0 -- that one is already merged to main. so we should be able to delete `sql_decorater_legacy` file I think.